src/main/java/it/eng/idsa/businesslogic/service/impl/KeystoreProvider.java

```diff
@@ -89,21 +91,29 @@ public KeystoreProvider(@Value("${application.targetDirectory}") Path targetDire
 				trustFactory.init(trustManagerKeyStore);
 				jksTrustStoreInputStream.close();
 			}
-			
+
 			jksKeyStoreInputStream.close();
-		} catch (IOException | KeyStoreException | NoSuchAlgorithmException | CertificateException | UnrecoverableKeyException e) {
-			logger.error("Error while loading keystore and truststore, {}", e);
+		} catch (IOException | KeyStoreException | NoSuchAlgorithmException | CertificateException
+				| UnrecoverableKeyException e) {
+			if (isEnabledDapsInteraction) {
+				logger.error("Error while loading keystore and truststore, {}", e);
+			} else {
+				logger.info("**********************************************************************");
+				logger.info("DAPS Interaction disabled. KeyStore or trustStore not correctly loaded");
```

Comment:
Can we change following information message to something like:
DAPS Interaction disabled. Keystore and/or truststore not loaded.
To remove this "correctly", since it might be misleading, that it was loaded but with some "magic" values